tests/plugins/os/windows/log/test_evt.py

```diff
@@ -110,3 +110,128 @@ def test_evt_direct_mode() -> None:
     records = list(target.evt())
 
     assert len(records) == 5
+
+
+def test_evt_time_change_warning_logged():  
+    """Test that time change events trigger warning logs in EVT plugin."""
+    from unittest.mock import Mock
+    
+    # Create a mock EVT record that represents a time change event
+    mock_record = Mock()
+    mock_record.EventID = 1
+    mock_record.SourceName = "Microsoft-Windows-Kernel-General"
+    mock_record.TimeGenerated = "2023-01-01T00:00:00Z"
+    mock_record.TimeWritten = "2023-01-01T00:00:00Z"
+    mock_record.EventCode = 1
+    mock_record.EventFacility = 0
+    mock_record.EventCustomerFlag = 0
+    mock_record.EventSeverity = 0
+    mock_record.EventType = 0
+    mock_record.EventCategory = 0
+    mock_record.Computername = "TestComputer"
+    mock_record.Strings = []
+    mock_record.Data = b""
+    
+    target = Target()
+    
+    # Mock the target's log warning method
+    original_warning = target.log.warning
+    warning_calls = []
+    def mock_warning(*args, **kwargs):
+        warning_calls.append((args, kwargs))
+        return original_warning(*args, **kwargs)
+    
+    target.log.warning = mock_warning
```

Comment:
Bro what are you doing, just use `caplog`. See `tests/loaders/test_vmx.py` for an example on how to use it on a `target.log`.